rpcserver.go

```diff
@@ -876,11 +914,47 @@ func handleEstimateFee(s *rpcServer, cmd interface{}, closeChan <-chan struct{})
 	return float64(feeRate), nil
 }
 
+// handleDelMiningAddr implements the delminingaddr command.
+func handleDelMiningAddr(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	c := cmd.(*btcjson.DelMiningAddrCmd)
+
+	// Decode the addresses encoded as a string into an in-memory object.
+	addr, err := btcutil.DecodeAddress(c.Address, activeNetParams.Params)
+	if err != nil {
+		return nil, &btcjson.RPCError{
+			Code:    btcjson.ErrRPCInvalidAddressOrKey,
+			Message: "Invalid address or key: " + err.Error(),
+		}
+	}
+
+	// Ensure the the passed address belongs to the same network that btcd
+	// is currently on.
+	if !addr.IsForNet(activeNetParams.Params) {
+		return nil, &btcjson.RPCError{
+			Code: btcjson.ErrRPCInvalidAddressOrKey,
+			Message: fmt.Sprintf("Mining address %v is on the "+
+				"wrong network: ", activeNetParams.Params),
```

Comment:
Same comments as above, use `fmt.Sprintf` for line 926 and modify the `Message` to be more informative in line 935 and 936